src/index.ts

```diff
@@ -9,9 +9,9 @@ import bezier from 'bezier-easing';
 import {DomUtil, LatLng, Layer, LayerOptions} from 'leaflet';
 import './draggable-patch';
 
-interface HarpLeafletOptions extends LayerOptions, MapViewOptions {
+type HarpLeafletOptions = Omit<LayerOptions & MapViewOptions, "canvas"> & {
     [key: string]: any;
```

Comment:
`[key: string]: any;` is no longer needed as we rely completely on `LayerOptions & MapViewOptions`

Reply:
done